index.js

```diff
@@ -220,7 +222,15 @@ async function run() {
       path.join(process.env.GITHUB_WORKSPACE, taskDefinitionFile);
     const fileContents = fs.readFileSync(taskDefPath, 'utf8');
     const taskDefContents = removeIgnoredAttributes(cleanNullKeys(yaml.parse(fileContents)));
-    const registerResponse = await ecs.registerTaskDefinition(taskDefContents).promise();
+    let registerResponse;
+    try {
+      registerResponse = await ecs.registerTaskDefinition(taskDefContents).promise();
+    } catch (error) {
+      core.setFailed("Failed to register task definition in ECS: " + error.message);
```

Comment:
Can you add a unit test to test registerTaskDefinition failures?

Reply:
Done